src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -197,14 +197,14 @@ const ContactDetails = ({ formikProps }: ContactDetailsProps) => {
           </HelpText>
           <Field
             as={RadioField}
-            checked={values.governanceTeams.isPresent === true}
+            checked={values.governanceTeams.isPresent}
```

Comment:
We actually need the `=== true` here because it is a controlled component.
https://fb.me/react-controlled-components

You can see the error in the dev tools when you load up the page and click `Yes`